packages/commercetools/theme/components/Checkout/VsfShippingProvider.vue

```diff
@@ -0,0 +1,239 @@
+<template>
+  <div class="shipping-provider">
+    <SfHeading
+      :level="3"
+      :title="$t('Shipping method')"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <div class="form">
+      <div v-if="error.load">
+        {{ $t('There was some error while trying to fetch shipping methods. We are sorry, please try with other shipping details or later.') }}
+      </div>
+      <div v-else-if="error.save">
+        {{ $t('There was some error while trying to select this shipping method. We are sorry, please try with other shipping method or later.') }}
+      </div>
+      <div class="form__radio-group">
+          <SfRadio
+            v-for="item in shippingMethods"
+            :key="item.id || item.name"
+            :label="item.name"
+            :value="item.id"
+            :selected="chosenShippingMethod.id"
+            @input="handleMethodSubmit(item)"
+            name="shippingMethod"
+            :description="item.description"
+            class="form__radio shipping"
+          >
+            <template #label="{ label }">
+              <div class="sf-radio__label shipping__label">
+                <div>{{ label }}</div>
```

Comment:
I'm not 100% sure about this, but maybe `label` should be translated?

Reply:
As both `label` and `description` comes from backend. I think the backend (e.g. CT) should provide them in a proper language. As far as I remember it does it for other entities like product. Wdyt @andrzejewsky ?